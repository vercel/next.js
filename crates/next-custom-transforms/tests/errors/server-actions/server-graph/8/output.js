/* __next_internal_action_entry_do_not_use__ {"006a88810ecce4a4e8b59d53b8327d7e98bbf251d7":"$$RSC_SERVER_ACTION_0"} */ import { registerServerReference } from "private-next-rsc-server-reference";
import { encryptActionBoundArgs, decryptActionBoundArgs } from "private-next-rsc-action-encryption";
export const $$RSC_SERVER_ACTION_0 = async function foo() {
    'use strict';
};
const foo = registerServerReference($$RSC_SERVER_ACTION_0, "006a88810ecce4a4e8b59d53b8327d7e98bbf251d7", null);
const bar = async ()=>{
    const x = 1;
    // prettier-ignore
    'use server';
};
