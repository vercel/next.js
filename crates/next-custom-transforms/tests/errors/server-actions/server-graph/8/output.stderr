  x The "use server" directive must be at the top of the function body.
  | 
    ,-[input.js:10:1]
  9 |   // prettier-ignore
 10 |   'use server'
    :   ^^^^^^^^^^^^
 11 | }
    `----
